server/server.go

```diff
@@ -57,6 +66,88 @@ func (server Impl) mainEndpoint(writer http.ResponseWriter, request *http.Reques
 	io.WriteString(writer, "Hello world!\n")
 }
 
+func (server Impl) listOfOrganizations(writer http.ResponseWriter, request *http.Request) {
+	organizations, err := server.Storage.ListOfOrgs()
+	if err != nil {
+		log.Println("Unable to get list of organizations", err)
+		responses.SendInternalServerError(writer, err.Error())
+	} else {
+		responses.SendResponse(writer, responses.BuildOkResponseWithData("organizations", organizations))
+	}
+}
+
+func (server Impl) readOrganizationID(writer http.ResponseWriter, request *http.Request) (storage.OrgID, error) {
+	organizationIDParam, found := mux.Vars(request)["organization"]
+
+	if !found {
+		const message = "Organization ID is not provided"
+		log.Println(message)
+		responses.SendInternalServerError(writer, message)
```

Comment:
Shouldn't it be "Bad Request" error?

Reply:
@Sergey1011010 as we discussed, in theory it should not happen that Gorilla mux calls this method w/o organization ID. I'll update comments accordingly

Reply:
it is commented, please look at new change set